pom.xml

```diff
@@ -48,6 +48,11 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-security</artifactId>
     </dependency>
+    
```

Comment:
I'm not sure caching is a must in such case. It's useful to manage eviction (with TTL for example) but the embedded forms are tightly coupled with the process definition and not subject to change. What would it bring ?